vulture/core.py

```diff
@@ -384,7 +389,11 @@ def visit_ClassDef(self, node):
 
     def visit_FunctionDef(self, node):
         for decorator in node.decorator_list:
-            if getattr(decorator, 'id', None) == 'property':
+            decorator = decorator.func if isinstance(
+                decorator, ast.Call) else decorator
+            name = decorator.attr if isinstance(
+                decorator, ast.Attribute) else decorator.id
```

Comment:
It would be helpful if we could use standard if-else statements here to see that all lines are tested.

Reply:
yes, that would be helpful coverage wise. Also, it helped me uncover a foolish bug -  that I forgot to ignoring the function in this branch. 😅 

Fix arriving.